matlab/MeCorr/MATLAB_CODE_methyl.m

```diff
@@ -106,9 +103,9 @@
     end
     
     acet_screen_rpmi(1,:) = NaN;
-
-     [sx spos] = sort(acet_screen_rpmi(:,1));
- acetgenessorted = unqgenes(spos);
+    
+    [sx spos] = sort(acet_screen_rpmi(:,1));
+    acetgenessorted = unqgenes(spos);

```

Comment:
change variable names to something more generic since this code looks at methylation now